test/run_test

```diff
@@ -36,14 +36,12 @@ test -n "${IMAGE_NAME-}" || false 'make sure $IMAGE_NAME is defined'
 test -n "${VERSION-}" || false 'make sure $VERSION is defined'
 test -n "${OS-}" || false 'make sure $OS is defined'
 
-CIDFILE_DIR=$(mktemp --suffix=postgresql_test_cidfiles -d)
+CID_FILE_DIR=$(mktemp --suffix=postgresql_test_cidfiles -d)
```

Comment:
Why this is needed?

Reply:
This does not break functionality. And only cosmetic issue does not make sense to me.